src/main/java/seedu/duke/Duke.java

```diff
@@ -87,10 +92,12 @@ private void runUntilExit() {
                 storage.save(trackingList);
                 ui.printReaction(commandOutput);
             } catch (PersonNotFoundException pnfe) {
-                System.out.println("Person not found!");
+                //System.out.println("Person not found!");
+                ui.notifyErrorToUser(pnfe);
```

Comment:
it's better to not leave commented code if you are unlikely to use it again

Reply:
Yes I agree too. Likewise for line 70 and 99 in Duke class

Reply:
I wanted to ask you guys first before I deleted the line. On whether I should keep the previous one or current one. 
If if looks ok I will delete the comments in the a new PR